nixos/modules/services/web-apps/bookstack.nix

```diff
@@ -16,9 +16,12 @@ let
 
   artisan = "${cfg.package}/artisan";
 
-  env-file-values = lib.attrsets.mapAttrs' (
-    n: v: lib.attrsets.nameValuePair (lib.strings.removeSuffix "_FILE" n) v
-  ) (lib.attrsets.filterAttrs (n: v: lib.strings.hasSuffix "_FILE" n) cfg.settings);
+  env-file-values =
+    with lib.attrsets;
```

Comment:
Please avoid `with` in front of such expressions. It is not easy for static analyzers and also very hard in case of refactor.